functions.js

```diff
@@ -3,15 +3,10 @@ document.querySelector(".c-nav__burger").addEventListener("click", () => {
   navInner.classList.toggle("c-nav__inner--active");
 });
 
-$(function() {
-  var nav = $(".c-nav");
-  $(window).scroll(function() {
-    var scroll = $(window).scrollTop();
+const nav = document.querySelector(".c-nav");
 
-    if (scroll >= 20) {
-      nav.addClass("c-nav--alt");
-    } else {
-      nav.removeClass("c-nav--alt");
-    }
-  });
+window.addEventListener("scroll", () => {
+  window.scrollY >= 20
```

Comment:
This is nice. Whoever thought of writing it like this is a genius. And really handsome.

Reply:
Aw thanks